tokio-util/tests/time_delay_queue.rs

```diff
@@ -778,6 +778,7 @@ async fn compact_change_deadline() {
     assert!(entry.is_none());
 }
 
+#[cfg_attr(target_os = "wasi", ignore = "FIXME: Does not seem to work with WASI")]
 #[tokio::test(start_paused = true)]
 async fn remove_after_compact() {
     let now = Instant::now();
```

Comment:
Because it uses `std::panic::catch_unwind()`, which isn't supported by Wasi.

Reply:
Many of the tests disabled for Wasi just have `#[cfg(not(target_os = 'wasi'))]`. Should the disabled tests have that, or have an ignore message? Just thinking about consistency.

Reply:
Prefer using ignore, but that doesn't always work because it might call methods that are missing. In that case just use `#[cfg(not(...))]`. I would prefer a comment explaining why in both cases.